include/AudioEngine.h

```diff
@@ -355,6 +357,11 @@ class LMMS_EXPORT AudioEngine : public QObject
 	AudioDevice * tryAudioDevices();
 	MidiClient * tryMidiClients();
 
+	inline const AudioDevice* audioDev() const
```

Comment:
```suggestion
	const AudioDevice* audioDev() const
```
This is already implicitly `inline`

Reply:
Done with commit c1e5de5b3b9.